src/main/resources/application.yaml

```diff
@@ -62,4 +62,5 @@ tasks:
   # Interval is in milliseconds.
   upload-letters: ${UPLOAD_LETTERS_INTERVAL:30000}
   mark-letters-posted: ${FTP_REPORTS_CRON:0 30 0 * * *}
+  stale-letters-report: ${STALE_LETTERS_REPORT_CRON:0 30 1 * * *}
```

Comment:
Would be good to run this in our e2e test as well by adding a line to the integrationTest application.yaml.

We could run it every second like we do for the mark letters posted task -   ${FTP_REPORTS_CRON:*/1 * * * * *}